lib/deserialize.js

```diff
@@ -49,6 +49,16 @@ module.exports = function (app, options) {
       ctx.args.filter.include = include;
     }
 
+    if (RelUtils.isJSONAPIPagination(ctx.req.query)) {
+      ctx.res.set({'Content-Type': 'application/vnd.api+json'});
```

Comment:
@digitalsadhu I think we should address setting content type somewhere else so it shouldn't be in this PR.